apollo-gradle-plugin/src/main/groovy/com/apollographql/apollo/gradle/ApolloExtension.groovy

```diff
@@ -46,7 +46,7 @@ class ApolloExtension {
     outputPackageName = project.objects.property(String.class)
     outputPackageName.set("")
 
-    customTypeMapping = project.objects.property(Map.class)
+    customTypeMapping = project.objects.mapProperty(String.class, String.class)
```

Comment:
I believe these were included in Gradle 4.x. Readme says 4.3 is required. Would be good to double check if 4.3 is fine or need to update the requirement in the Readme.

Reply:
Can check, but also  feel like only supporting current major version should be fine too. Gradle is already on 5.5, and these APIs are also not stable yet

Reply:
My Sample changes is merged. If you update your branch then you will have the sample. Sample now uses a separate project with composite builds. We can quickly try with different set of Gradle versions there. By changing the wrapper on the sample.